### PR TITLE
count number of pending datasets - fixes #22

### LINKED ISSUE #22: Show count of pending requests in dashboard tab
screenshot_2018-12-10 manage - org_admin - users - ckan

The easiest way is to create a helper for the pending datasets query (which will be reused in the controller) and call it from `dashboard.html`

In addition, it would be nice to update the counter on the header dashboard icon as well:

screenshot_2018-12-10 datasets - ckan

Currently this only refers to new activities on datasets or orgs you follow, but we could add the pending requests as well. To do so, we need to override [this bit](https://github.com/ckan/ckan/blob/master/ckan/templates/header.html#L21:L29) with our custom helper. Our helper will add the result of `h.new_activities()` plus the count of pending requests returned by the first helper mentioned above.

### PR DESCRIPTION
**Pull request by zelima**
image

### CODE REVIEW

File: ckanext/ed/templates/user/dashboard.html [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {% extends "user/edit_base.html" %}
 
 {% set user = c.userobj %}
+{% set pending_datasets = h.ed_get_pending_datasets(c.user) %}
```
Comment (amercader) [inline ckanext/ed/templates/user/dashboard.html]: Do we need this one now that we have the same call on `header.html`

Comment (zelima) [reply ckanext/ed/templates/user/dashboard.html]: @amercader strange but yes, it's not accessible from dashboard - showing 0

Comment (amercader) [reply ckanext/ed/templates/user/dashboard.html]: ok

Comment (amercader) [reply ckanext/ed/templates/user/dashboard.html]: This is some Jinja2 scope issue

**Pull request merged**